src/data/faq.json

```diff
@@ -293,12 +293,15 @@
                         ]
                     },
                     {
-                        "title": "Can I use a complete vaccination status in the app as a travel permission?",
+                        "title": "Can I use a vaccination certificate in the app as a travel permission?",
                         "anchor": "vac_cert_privileges_travel",
                         "active": true,
                         "textblock": [
-                            "For example, whether you are allowed to travel with completed vaccination protection, depends on the regulations of the respective country.",
-                            "Among other things, you can use this   <a href='https://reopen.europa.eu/en' target='_blank' rel='noopener noreferrer'>tool</a> in order to plan your journey within the EU."
+                            "The digital certificate of vaccination of the EU can not only be used in Germany. The digital certificate of vaccination of the EU issued in Germany can also be regarded as proof of vaccinations in other EU countries if the respective EU member state allows digital proof. The same applies to recovery certificates and test certificates. From Corona-Warn-App version 2.6, the validity of your certificate can also be checked within the app before departure.",

```

Comment:
Maybe use "from the EU" and not "of the EU" to avoid writing "of" 2 times?

Reply:
"from the EU" doesn't sound right to me, but I understand the problem with "of the EU" occuring twice. How about "EU digital vaccination certificate"?

Reply:
Yes, sounds good, in the app it's called "EU digital COVID vaccination certificate", maybe go with this term?

Reply:
Sounds good, I'll go with that one. Thanks!